burr/core/graph.py

```diff
@@ -165,10 +173,49 @@ def visualize(
                 label=condition.name if include_conditions and condition is not default else None,
                 style="dashed" if transition.condition is not default else "solid",
             )
+
+        # the default format is png if nothing else is specified
+        render_kwargs = {"format": "png", "view": view}
         if output_file_path:
-            digraph.render(output_file_path, view=view)
+            output_file_path = pathlib.Path(output_file_path)
```

Comment:
It might be nice for this path stuff to be a separate function --- it's a bit complex/inline